### PR TITLE
remove printfs from crypto tests

### PR DESCRIPTION
**Pull request by bvavala**
Signed-off-by: Bruno Vavala <bruno.vavala@intel.com>

### CODE REVIEW

File: common/tests/testCrypto.cpp [hunk 1]
```diff
@@ -43,6 +41,20 @@ namespace constants = pdo::crypto::constants;
...
+    vsnprintf_s(buf, BUFSIZ, fmt, ap);
+    va_end(ap);
+#if _UNTRUSTED_
+    pdo::logger::Log((pdo_log_level_t)level, buf);
+#else
+    ocall_Log(level, buf);
+#endif
+}  // Log
+
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Comment (cmickeyb) [inline common/tests/testCrypto.cpp]: Why not just use the functions in log.cpp? You would basically just call pdo::logger::SetLogFunction(ocall_Log) if trusted and pdo:logger::SetLogFunction(pdo::logger::LogStdOut) if untrusted? I think we can just wrap the printf version of the function with the UNTRUSTED wrapper and the rest should work. That is... making a single copy of Log work would be a much better solution than implementing yet another version.

File: common/tests/trusted/app/TestApp.cpp [hunk 2]
```diff
@@ -25,6 +25,18 @@
...
+
+void Log(int level, const char* fmt, ...)
+{
+    char buf[BUFSIZ] = {'\0'};
+    va_list ap;
+    va_start(ap, fmt);
+    vsnprintf_s(buf, BUFSIZ, fmt, ap);
+    va_end(ap);
+    ocall_Log(level, buf);
+}  // Log
```
Comment (cmickeyb) [inline common/tests/trusted/app/TestApp.cpp]: Again... YET ANOTHER IMPLEMENTATION.

File: common/tests/testCrypto.cpp (hunk 1 above)
Comment (bvavala) [reply common/tests/testCrypto.cpp]: done

File: common/tests/trusted/app/TestApp.cpp (hunk 2 above)
Comment (bvavala) [reply common/tests/trusted/app/TestApp.cpp]: removed

**Pull request merged**